Hackney.Shared.HousingSearch.Tests/Models/TempAccommodationInfoTests.cs

```diff
@@ -21,6 +21,7 @@ public void DomainTempAccommodationInfoHasPropertiesSet()
 
             //assert
             createdDomaninTAInfo.BookingStatus.Should().Be(aQueryableTAInfo.BookingStatus);
+            createdDomaninTAInfo.AssignedOfficer.Should().Be(aQueryableTAInfo.AssignedOfficer);
```

Comment:
This will compare object references.
Try using `BeEquivalent` for the recursive check of data nested within.

Reply:
Addressed in [cebad9e](https://github.com/LBHackney-IT/housing-search-shared/pull/74/commits/cebad9ecced7120c18e3bbcd9cd2d26a571dc601)